digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -203,6 +203,9 @@ private void runWithWorkspace(Path projectPath, TaskRequest request)
         catch (RuntimeException ex) {
             throw new RuntimeException("Failed to process variables", ex);
         }
+        catch (AssertionError ex) {
```

Comment:
ditto

Reply:
In this case we can catch the AssertionError because of ConfigEvalEngine(it calls nashorn).

Reply:
If we don't catch AssertionError here, it will be caught by above modification. So this fix is not required. But I guess by catching ConfigEvalEngine related AssertionError in this place may make clear the cause. So I also catch it.